projects/packages/search/build/webpack.instant.config.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const path = require( 'path' );
+const {
+	defaultRequestToExternal,
+	defaultRequestToHandle,
+} = require( '@wordpress/dependency-extraction-webpack-plugin/lib/util' );
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+/**
+ * Internal dependencies
+ */
+const definePaletteColorsAsStaticVariables = require( './define-palette-colors-as-static-variables' );
+const AddReadableJSAssetsPlugin = require( './add-readable-js-assets' );
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+const baseWebpackConfig = getBaseWebpackConfig(
+	{ WP: false },
+	{
+		entry: {
+			main: path.join( __dirname, '../modules/search/instant-search/loader.js' ),
+		},
+		// Putting a cache buster in the query string is not documented, but confirmed by the author of Webpack.
+		// `But better use the hash in filename and use no query parameter.`
+		// The reason probably is because it's not the best way to do cache busting.
+		// More information: https://github.com/webpack/webpack/issues/2329
+		'output-chunk-filename': 'jp-search.chunk-[name].[contenthash:20].min.js',
+		'output-filename': 'jp-search-[name].bundle.min.js',
+		'output-path': path.join( __dirname, '../_inc/build/instant-search' ),
```

Comment:
Just a thought, could we allow an env variable to set the output path, with a default location set to the package? So that we could keep the build at the original place as before.

Reply:
I'm not sure doing so will be worth the increased build complexity once we have the standalone plugin up and running; we would need to output built assets to two directories and add configurability to the Instant Search class to accommodate enqueueing from different directories. 

I propose that we contain all parts of the build process (Webpack build and subsequent enqueuing of built assets) in the package for simplicity.